controlplane/pkg/nsmd/utils.go

```diff
@@ -64,3 +67,13 @@ func GetLocalIPAddress() string {
 	}
 	return "127.0.0.1"
 }
+
+// RandomHexString - generates random string of desired length
+func RandomHexString(size int) string {
```

Comment:
Can we use UID here?

Reply:
Could you please elaborate what UID can be used here, and how?
Do you mean the function should generate a string in form of a UUID (and named accordingly)?
Or github.com/google/uuid should be used instead of RandomHexString?

Reply:
>Do you mean the function should generate a string in form of a UUID (and named accordingly)?
Yes, I mean use UUID instead of `RandomHexString`.

Reply:
- [ ] use UUID string instead of RandomHexString